### DISCUSSION TITLE
Allow login without a GitHub account

### DISCUSSION THREAD

**Discussion by AaronM04** [rust-lang/crates.io#9446] [General]
Hello crates.io peeps,

Would you all be open to a different login method besides a GitHub account? Doing so would reduce the lock-in to GitHub (owned by Microsoft), which seems it would be a very positive change to me.

I would be happy to beta test such a solution!

Best,
Aaron

**Comment by Turbo87** [comment]
see https://github.com/rust-lang/crates.io/issues/326
